bahatii/Property-Pro-lite
# finishes #167094891

#### What does this PR do?
it enable  the user to sign up,

#### Description of Task to be completed?
create a page,
add a sign up form

#### How should this be manually tested?
the user should fill in the form, the required information and the click to the sign up button
and automatically he should be required to sign in
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#167094891

#### Screenshots (if appropriate)
N/A
#### Questions:
